activesupport/lib/active_support/core_ext/range/compare_range.rb

```diff
@@ -51,31 +51,6 @@ def include?(value)
         super
       end
     end
-
-    # Extends the default Range#cover? to support range comparisons.
-    #  (1..5).cover?(1..5)  # => true
-    #  (1..5).cover?(2..3)  # => true
-    #  (1..5).cover?(1...6) # => true
-    #  (1..5).cover?(2..6)  # => false
-    #
-    # The native Range#cover? behavior is untouched.
-    #  ('a'..'f').cover?('c') # => true
-    #  (5..9).cover?(11) # => false
-    #
-    # The given range must be fully bounded, with both start and end.
-    def cover?(value)
```

Comment:
Can you also remove the range comparison tests?

https://github.com/rails/rails/blob/507c1895a43d3d0bc4bc8fb111058ddac9a99635/activesupport/test/core_ext/range_ext_test.rb#L154-L181

Reply:
Sure, I left them for review purposes, I've removed it.